packages/ui/src/common/helpers.ts

```diff
@@ -3,7 +3,7 @@ export const capitalizeFirstLetter = <T extends string>(str: T) =>
 
 export const lowerFirstLetter = (str: string): string => str.charAt(0).toLowerCase() + str.slice(1)
 export const camelCaseToText = (str: string): string => {
-  return capitalizeFirstLetter(str.replace(/(?<=[a-z])([A-Z])|(?<=[A-Z])([A-Z][a-z])/g, ' $1$2'))
+  return capitalizeFirstLetter(str.replace(/([A-Z]+)*([A-Z][a-z])/g, '$1 $2'))
```

Comment:
Wow Safari only crashes because of a regexp problem :open_mouth: 
The new regexp mostly works but it adds an extra space at the beginning of the string (and also ideally we should add tests to these utility functions to make their expected behavior clearer).
IMO since there's already some lodash dependencies in the `ui` workspace the best, would be to remove this function and just use the [lodash `startCase` function](https://lodash.com/docs/4.17.15#startCase) instead. Also `lodash@^4` should be added to the `ui/package.json` (ATM the projects only builds because it's a dependency of `@/joystream/types`)